packages/grafana-data/src/types/theme.ts

```diff
@@ -240,6 +240,10 @@ export interface GrafanaTheme extends GrafanaThemeCommons {
     formCheckboxBgChecked: string;
     formCheckboxBgCheckedHover: string;
     formCheckboxCheckmark: string;
+
+    // Active Tab Header Color
+    tabHeaderGradientStart: string;
+    tabHeaderGradientEnd: string;
```

Comment:
Can you please remove those variables from the theme and just use correct theme colors directly in the Tab component?

Also pinging @jessover9000 - what would be the theme colors to use for dark theme's Tab instead of currently used  `#f05a28 30%, #fbca0a 99%`?

Reply:
Thank you for asking! Since red and orange are hard to see on black for certain types of color blindness and consistent high contrast can't hurt, for the dark theme I would suggest choosing the lightest orange from the accessible color palette that @Ijin08 defined (#ffb375) in stead of the brand orange. In the light theme, we can keep using our currently used brand orange (#f05a28). The yellow can stay the same in both light and dark (#ffee52). I haven't really played around with the position of the colors in the gradient, I guess we can keep 30% and 99% for minimal change.

Reply:
@dprokop @jessover9000 Thanks for your feedback. I am not yet completely familiar with the codebase but currently I can't find none of the mentioned colors in the theme palettes.

Colors used in dark theme:
```
yellow: '#ecbb13',
orange: '#eb7b18',
orangeDark: '#ff780a',
```

Colors used in light theme:
```
yellow: '#ff851b',
orange: '#ff7941',
orangeDark: '#ed5700',
```

I would not like to change these values because it might break other component styles that I am not aware of.

Reply:
@jessover9000 sorry for not being clear in my question. It was more about what colors from our palette should be used instead of the ones used currently? We probably should not introduce new colors but either use the ones we have or adjust them to the needs of the Tab component. I would opt for reusing existing colors TBH.

Reply:
@dprokop @jessover9000 Could you clarify the colors?

Reply:
ah, yes, sorry for the delay. this issue keeps recurring, we need to fix up the grafana-ui color palette so that we're all using the same colors. 🤪 In the meantime, I guess we could use a gradient from the values @twei55 mentioned for dark theme yellow and orange and exclude orange-dark, to give more color contrast.